website/content/docs/templates/json_to_hcl.mdx

```diff
@@ -0,0 +1,157 @@
+---
+description: |
+  This section highlights the differences between JSON and HCL2 templates, and explains the parallels between the two specifications.
+page_title: JSON to HCL
+---
+
+# Packer templates
+
+Packer uses templates to orchestrate builds for one or multiple images.
+In JSON, you would declare a series of builders, provisioners and post-processors to build images.
+In HCL2, things are different, as the configuration language allows you to specify builders
+through sources, and weave them in build blocks.
```

Comment:
What do you think about mentioning JSON as a legacy template format? 
```suggestion
In JSON, considered legacy, you would declare a series of builders, provisioners,  and post-processors to build images. In HCL2, things are different, as the configuration language allows you to specify builders
through sources and weave them in build blocks.
```

Reply:
I'm not sure about the "considered legacy" phrasing, but we can highlight the fact that JSON is indeed deprecated here yes.

Reply:
That's fair. There is a not at the top of the JSON templates document https://developer.hashicorp.com/packer/docs/templates/legacy_json_templates maybe having a link to the JSON page.